client/public/static/locales/pt/auth.json

```diff
@@ -1,44 +1,44 @@
-  "CHANGE_PASSWORD": "__STRING_NOT_TRANSLATED__",
-  "CONFIRM_PASSWORD": "__STRING_NOT_TRANSLATED__",
-  "DISPLAY_NAME": "__STRING_NOT_TRANSLATED__",
-  "EMAIL": "__STRING_NOT_TRANSLATED__",
-  "FORGOT_PASSWORD": "__STRING_NOT_TRANSLATED__",
-  "LOGIN": "__STRING_NOT_TRANSLATED__",
-  "LOGOUT": "__STRING_NOT_TRANSLATED__",
-  "NEW_PASSWORD": "__STRING_NOT_TRANSLATED__",
-  "NO_ACCOUNT": "__STRING_NOT_TRANSLATED__",
-  "NO_ACCOUNT_SIGNUP": "Don't have an account? <1>Sign up here.</1>",
-  "OLD_PASSWORD": "__STRING_NOT_TRANSLATED__",
-  "PASSWORD": "__STRING_NOT_TRANSLATED__",
-  "PASSWORD_CHANGE_SUCCESS": "__STRING_NOT_TRANSLATED__",
-  "REMEMBER_ME": "__STRING_NOT_TRANSLATED__",
-  "REQUEST_PASSWORD_RESET": "__STRING_NOT_TRANSLATED__",
+  "ALREADY_HAVE_ACCOUNT": "Já tem uma conta? <1>Faça o login aqui.</1>",
+  "CHANGE_PASSWORD": "Mudar senha",
+  "CONFIRM_PASSWORD": "Confirmar senha",
+  "DISPLAY_NAME": "Nome de exibição",
+  "EMAIL": "Email",
+  "FORGOT_PASSWORD": "Esqueceu sua senha?",
+  "LOGIN": "Login",
+  "LOGOUT": "Logout",
+  "NEW_PASSWORD": "Nova senha",
+  "NO_ACCOUNT": "Não tem uma conta? Registe-se aqui.",
+  "NO_ACCOUNT_SIGNUP": "Não tem uma conta? <1>Registe-se aqui.</1>",
+  "OLD_PASSWORD": "Senha antiga",
+  "PASSWORD": "Senha",
+  "PASSWORD_CHANGE_SUCCESS": "Sua senha foi alterada com sucesso.",
+  "REMEMBER_ME": "Lembrar de mim na minha próxima visita",
+  "REQUEST_PASSWORD_RESET": "Solicitar redefinição de senha",
   "REQUEST_PASSWORD_RESET_EMAIL_SENT": {
-    "DESCRIPTION": "__STRING_NOT_TRANSLATED__",
-    "TITLE": "__STRING_NOT_TRANSLATED__"
+    "DESCRIPTION": "Enviamos um email. Siga o link para redefinir sua senha.",
+    "TITLE": "Sucesso"
   },
-  "RESEND_VERIFICATION_EMAIL": "__STRING_NOT_TRANSLATED__",
-  "RESET_PASSWORD": "__STRING_NOT_TRANSLATED__",
+  "RESEND_VERIFICATION_EMAIL": "Reenviar email de verificação",
+  "RESET_PASSWORD": "Redefina sua senha",
   "RESET_PASSWORD_SUCCESS": {
-    "DESCRIPTION": "__STRING_NOT_TRANSLATED__",
-    "TITLE": "__STRING_NOT_TRANSLATED__"
+    "DESCRIPTION": "Sua senha foi alterada com sucesso. Redirecionando...",
+    "TITLE": "Sucesso"
   },
-  "SIGN_UP": "__STRING_NOT_TRANSLATED__",
-  "THANKS_FOR_SIGNING_UP": "__STRING_NOT_TRANSLATED__",
+  "SIGN_UP": "Registe-se",
+  "THANKS_FOR_SIGNING_UP": "Obrigado por se registar!",
   "VALIDATION": {
-    "CONSECUTIVE_SEPARATOR_RULE": "__STRING_NOT_TRANSLATED__",
-    "DISPLAY_NAME_REQUIRED": "__STRING_NOT_TRANSLATED__",
-    "DISPLAY_NAME_RULES": "__STRING_NOT_TRANSLATED__",
-    "EMAIL_REQUIRED": "__STRING_NOT_TRANSLATED__",
-    "PASSWORDS_DO_NOT_MATCH": "__STRING_NOT_TRANSLATED__",
-    "PASSWORD_REQUIRED": "__STRING_NOT_TRANSLATED__",
-    "PASSWORD_RULES": "__STRING_NOT_TRANSLATED__",
-    "REQUIRED_FIELD": "__STRING_NOT_TRANSLATED__",
-    "START_END_RULE": "__STRING_NOT_TRANSLATED__",
-    "VALID_EMAIL": "__STRING_NOT_TRANSLATED__"
+    "CONSECUTIVE_SEPARATOR_RULE": "Não pode conter pontos finais consecutivos ou sublinhados",
```

Comment:
I think I have translated a string like that before. I have not used "sublinhado" or "pontos finais consecutivos". It's not wrong but it's better decide what is gonna be shown for avoiding lack of pattern.